app/views/attachment_mailer/new_attachment_notification.html.haml

```diff
@@ -1,23 +1,160 @@
-%table.full-width.header
+%table.body
   %tr
-    %td.hero
-    %td
-      = image_tag "emails/icon-pencil-circle.png", class: "header-icon"
+    %td.center{:align => "center", :valign => "top"}
+      %center
+        / Start Header
+        %table.row.header
+          %tr
+            %td.center{:align => "center"}
+              %center
+                %table.container
+                  %tr
+                    %td.wrapper.last
+                      %table.twelve.columns
+                        %tr
+                          %td.sub-columns.center{:align => "center", :valign => "top"}
+                            %center
+                          %td.expander
+        %table.row
+          %tr
+            %td.center{:align => "center"}
+              %center
+                %table.container
+                  %tr
+                    %td.wrapper.last
+                      %table.twelve.columns
+                        %tr
+                          %td.sub-columns.center{:align => "center", :valign => "top"}
+                            %center
+                              = image_tag "emails/icon-pencil-circle.png", class: "no-float-image"
+                          %td.expander
+        
+          / End Header
```

Comment:
woah I didn't realize HAML had one slash for comments. so efficient with characters!